### PR TITLE
Prod Seed data

### PR DESCRIPTION
**Pull request by ShaydeNofziger**
This adds in a bunch of default data from the current version of the game. Mostly: Actions, TeamTypes, and Events.

NOTE: This changes the format of how we're doing seed data so data can be easily appended to a JSON array for production.

### CODE REVIEW

File: seeds/data/json/action.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
+const DefaultActions = [
```
Comment (jcdesimp) [inline seeds/data/json/action.js]: Any reason we don't just want this to be a JSON file?
JSON files can be required directly and just become an object.

File: seeds/data/json/event.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+const DefaultEvents = [
```
Comment (jcdesimp) [inline seeds/data/json/event.js]: same with this

File: seeds/data/json/teamtype.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+const DefaultTeamTypes = [
```
Comment (jcdesimp) [inline seeds/data/json/teamtype.js]: and this...

File: seeds/data/json/user.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+const DefaultUserData = [
```
Comment (jcdesimp) [inline seeds/data/json/user.js]: same..

**Review by jcdesimp — CHANGES_REQUESTED** [review]

File: seeds/data/json/action.js (hunk 1 above)
Comment (ShaydeNofziger) [reply seeds/data/json/action.js]: Oooh nice. Will do

File: seeds/data/json/teamtype.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
...ption": "You are developing a new line of mobile applications for the Rainforest system. These applications will be on Android devices. They interact primarily with the Rainforest API, but also cache important data such as payment data and various other personal information. These applications involve a combination of both native implementations and HTML5 implementations on mobile devices.\nUnlike the iOS team, this product is unique because it’s also open source. This means that you can leverage a community of volunteers to inspect your code.\nSpecifically, you are able to get code inspections of new development changes each month (#3), with no devcap cost (as a results you get the Hacker Mindset bonus too). Track this on your sheet by changing the Dev Capacity entry the Tasks page to zero.\nYour products will be considered mature at the beginning of Month 3.\nYour products are considered to be public-facing, meaning untrusted users outside the company interact with your product.\nYour product primarily interacts with API, with some interaction with Infrastructure and Web.\nGoal: devote at least 30 devcaps of Routine Development by the end of Month 5, or this team will be fired.",
```
Comment (ShaydeNofziger) [inline seeds/data/json/teamtype.json:12]: No clue as to why GitHub highlighting isn't working on some of these description fields. jsonvalidator says it's valid. My guess is the string length.

**Review by jcdesimp — APPROVED** [review]

**Pull request merged**